stdlib/sys/file.gr

```diff
@@ -577,6 +585,44 @@ provide let fdPread = (fd: FileDescriptor, offset: Int64, size: Number) => {
   return Ok((WasmI32.toGrain(strPtr): Bytes, tagSimpleNumber(nread)))
 }
 
+/**
+ * Get information about a preopened directory.
+ *
+ * @param fd: The file descriptor to check
+ * @returns `Ok(Dir{name})` if successful or `Err(exception)` otherwise
```

Comment:
```suggestion
 * @returns `Ok(Dir{prefix, fd})` if successful or `Err(exception)` otherwise
 *
 * @since v0.6.0
```
Need since docs and it looks like the Dir variant was doc'd wrong